lego-webapp/app/routes/hiddenAdmin/components/BannerEditor.tsx

```diff
@@ -0,0 +1,96 @@
+import { LoadingPage, Page } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Field } from 'react-final-form';
+import { Helmet } from 'react-helmet-async';
+import { useNavigate, useParams } from 'react-router';
+import { TextInput, Form, LegoFinalForm, CheckBox } from '~/components/Form';
+import { SubmitButton } from '~/components/Form/SubmitButton';
+import {
+  createBanner,
+  editBanner,
+  fetchBannerById,
+} from '~/redux/actions/BannerActions';
+import { useAppDispatch, useAppSelector } from '~/redux/hooks';
+import { CreateBanner } from '~/redux/models/Banner';
+import { selectBannerById } from '~/redux/slices/banner';
+
+import { guardLogin } from '~/utils/replaceUnlessLoggedIn';
+
+type BannerEditorParams = {
+  bannerId?: string;
+};
+const BannerEditor = () => {
+  const { bannerId } = useParams<BannerEditorParams>();
+
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  usePreparedEffect(
+    'fetchBannerForEditor',
+    () => bannerId && dispatch(fetchBannerById(bannerId)),
+    [bannerId],
+  );
+
+  const isNew = !bannerId;
+  const banner = useAppSelector((state) => selectBannerById(state, bannerId));
+
+  if (!isNew && !banner) {
+    return <LoadingPage loading />;
+  }
+
+  const onSubmit = (data: CreateBanner) => {
+    const action = isNew ? createBanner(data) : editBanner(data, bannerId);
+    dispatch(action).then(() => navigate('/sudo/banners'));
+  };
+
+  return (
+    <Page
+      title={isNew ? 'Nytt Banner' : `Redigerer: ${banner?.header}`}
+      back={{ href: '/sudo/banners' }}
+    >
+      <Helmet title={'Rediger Bannere'} />
```

Comment:
```suggestion
  const title = isNew ? 'Nytt Banner' : `Redigerer: ${banner?.header}`

  return (
    <Page
      title={title}
      back={{ href: '/sudo/banners' }}
    >
      <Helmet title={title} />
```

The current helmet title will be wrong when not editing